src/__tests__/Divider.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import 'jest-styled-components'
+import { Divider, theme } from '..'
+
+describe('Divider', () => {
+  test('renders', () => {
+    const json = renderer.create(<Divider />).toJSON()
+    expect(json).toMatchSnapshot()
+  })
+
+  test('width prop sets width', () => {
```

Comment:
You shouldn't have to write tests for `styled-system` too much since it's 100% covered